### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by 1Jayso** [OpenMined/courses#8] [Introduce Yourself]
Hello everyone I am Joseph from Ghana. Nice meeting you all

**Comment by ch3njust1n** [comment]
Hi, I'm Justin from Boston!

**Reply by 1Jayso** [reply]
Nice to meet you Justin :)

**Comment by yemikifouly** [comment]
@1Jayso @ch3njust1n Hello Joseph and Justin! 👋 I am Yemi from Benin! Glad to have you here! 🙂 I hope you enjoy the course!

**Reply by mcleonard** [reply]
Hi there Joseph and Justin! I'm Mat from San Francisco. Excited to help you learn some cooooool privacy tech

**Reply by 1Jayso** [reply]
Thank you Yemi and  Mat, I am excited to connect with you both. I look forward to an exciting learning experience!
